pkg/commands/auth/run.go

```diff
@@ -0,0 +1,77 @@
+package auth
+
+import (
+	"context"
+	"os"
+
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/cli/cli/config/types"
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/flag"
+	"github.com/aquasecurity/trivy/pkg/log"
+)
+
+func Login(_ context.Context, registry string, opts flag.Options) error {
+	if len(opts.Credentials) == 0 {
+		return xerrors.New("username and password required")
+	} else if len(opts.Credentials) > 1 {
+		return xerrors.New("multiple credentials are not allowed")
```

Comment:
`--password-stdin` doesn't support `Comma-separated`. But we can use `user` and `password`.
If we don't want use a multiple credentials - we need to say about that in docs:


 https://github.com/aquasecurity/trivy/blob/a69a936c37fa15bc28469fdb1e494ba5e549be3e/docs/docs/advanced/private-registries/index.md?plain=1#L41-L54

Reply:
Added https://github.com/aquasecurity/trivy/pull/7664/commits/6c44acd3ed50e1d8dcfd170b209b74ae79772376

Reply:
I think we don't need to split `--password-stdin` with `,` here:
https://github.com/aquasecurity/trivy/blob/b5be50fa81ae9d88573270a75cd17b8caf366944/pkg/flag/registry_flags.go#L86-L87
Without `strings.Split` we will immediately return an error on the number of users/passwords.

Reply:
@knqyf263 ^

Reply:
Thanks. I tested it locally and forgot to revert.